### PR TITLE
Pylint style fixes

### PR DESCRIPTION
**Pull request by ldoktor**
Addressing the newly added C0209 and W1514 checks failures.

Signed-off-by: Lukáš Doktor <ldoktor@redhat.com>

### CODE REVIEW

File: scripts/aexpect_helper [hunk 1]
```diff
@@ -119,11 +118,11 @@ def main():     # too-many-* pylint:disable=R0914,R0912,R0915
 
             # Write shell PID to file
             server_log.info('Writing shell PID file %s', shell_pid_filename)
-            with open(shell_pid_filename, "w") as file_obj:
+            with open(shell_pid_filename, "w", encoding="utf-8") as file_obj:
```
Comment (lmr) [inline scripts/aexpect_helper:121]: Ah, an option for open that I didn't know about. Living and learning.

**Review by lmr — APPROVED** [review]
Looks good, approved!

File: scripts/aexpect_helper (hunk 1 above)
Comment (ldoktor) [reply scripts/aexpect_helper:121]: Well in py2 one had to use codecs to open with encoding, in py3 they added it as an argument and in 3.10 they added `encoding="locale"` which should be what we use in avocado (`locale.getpreferredencoding()`)

**Comment by pevogam** [comment]
Great job @ldoktor, I was so behind with GitHub notifications that I couldn't see this on time. In any case thanks for this!

**Pull request merged**